lib/middleware/auth.js

```diff
@@ -13,19 +13,20 @@ function appAuth(req, res, next){
   var envAccessKey = req.get('x-fh-env-access-key');
 
   log.logger.debug("Authenticating App ", apiKey, envAccessKey, req.params);
+  // var fhDataman = true;
 
-  if(!apiKey || !envAccessKey){
-    res.status(401);
-    return res.end();
-  }
+  // if(!apiKey || !envAccessKey || !fhDataman){
+  //   res.status(401);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)